### DISCUSSION TITLE
Daly BMS - BLE connection

### DISCUSSION THREAD

**Discussion by drtinaz** [Louisvdw/dbus-serialbattery#606] [New BMS requests]
I see some work being done on the nightly's for the jkbms ble support. Is there any support in there or future planned support for the Daly?

**Comment by mr-manuel** [comment]
Not yet. I moved this to the BMS requests. So we can look at it, if there are enough upvotes.

**Reply by transistorgit** [reply]
If we're going to start  Daly BLE, it should be considered to also connect the active balancers. Probably quite a lot new users would buy also the active balancers, which are additional devices with own bluetooth connections. I wouldn't see much sense in doing this work and not supporting the balancers.

**Reply by drtinaz** [reply]
I second this. I also have the smart active balancer in addition to the bms. Would bt connection to the driver be the only connection to the bms? Meaning...while connected to the driver, there cannot be a connection to the mobile app?

**Comment by RodeWillem** [comment]
Also 5A active balancer

**Comment by Alestrix** [comment]
Is this still in the backlog or is someone already working on this? I'd be very interested!
